src/item_menu.c

```diff
@@ -2172,10 +2193,15 @@ static void SellItem(u8 taskId)
     s16 *data = gTasks[taskId].data;
     u16 *scrollPos = &gBagPosition.scrollPosition[gBagPosition.pocket];
     u16 *cursorPos = &gBagPosition.cursorPosition[gBagPosition.pocket];
+#if I_SELL_VALUE >= GEN_9
+    u32 sellValue = (ItemId_GetPrice(gSpecialVar_ItemId) / 4) * tItemCount;
+#else
+    u32 sellValue = (ItemId_GetPrice(gSpecialVar_ItemId) / 2) * tItemCount;
+#endif
```

Comment:
I think these would read a lot better with something like
```c
#if I_SELL_VALUE >= GEN_9
#define ITEM_SELL_FACTOR 4
#else
#define ITEM_SELL_FACTOR 2
#endif
```
or potentially
```c
static inline GetItemSellFactor(void)
{
    if (I_SELL_VALUE >= GEN_9)
        return 4;
    return 2;
}
```
which can then be used over the magic numbers without having preproc statements everywhere.

Reply:
I don't think we should add a function just for a config option so I'll do the first one.

Reply:
In this case I see the inline function more like a replacement for C23 `constexpr`.